test/functional/apps/visualize/_line_chart.js

```diff
@@ -99,10 +78,41 @@ bdd.describe('visualize app', function describeIndexTests() {
       });
     });
 
-    bdd.it('should show correct data', function pageHeader() {
 
-      var remote = this.remote;
-      var expectedChartData = ['jpg 9,109', 'css 2,159', 'png 1,373', 'gif 918', 'php 445'];
+    bdd.it('should show correct chart order by Term', function pageHeader() {
+
+      // this test only verifies the numerical part of this data
+      // https://github.com/elastic/kibana/issues/8141
+      var expectedChartData = ['png 1,373', 'php 445', 'jpg 9,109', 'gif 918', 'css 2,159'];
```

Comment:
Since we're only comparing against the numerical parts of these strings, why not either a) hardcode these values as numbers or b) precompute the numerical values ahead of time to improve readability?

a:
```javascript
const expectedChartsData = [1373, 445, 9109, 918, 2159];
```

b:
```javascript
const expectedChartData = ['jpg 9,109', 'css 2,159', 'png 1,373', 'gif 918', 'php 445'];
// Strip out label value and convert to numbers.
const expectedChartY = expectedChartData.map(val => val.split(' ')[1]).replace(',', ''));
```